typescript/tuple_tree.ts

```diff
@@ -39,6 +39,22 @@ export function safeParseInt(str?: string): number | undefined {
     }
 }
 
+/**
+ * Parses a bigint safely, returning undefined if the number cannot be represented as a decimal
+ * string
+ */
+export function safeParseBigInt(str?: string): bigint | undefined {
+    if (str !== undefined) {
```

Comment:
It sort-of hand-tailored to the use in `model.ts` where we do:
```
this.epoch = safeParseBigInt(match.groups["Epoch"]) || 0n;
```

Reply:
Can't you push `|| "0"` in?

Reply:
Having looked a bit closer to the regex that matches the `MetaAddress`, I've concluded that `BigInt` can be used directly because the content of the regex is very strict. I've updated the commit accordingly.